strawberryfields/gbs/similarity.py

```diff
@@ -32,9 +55,53 @@
     orbits
     orbit_cardinality
     event_cardinality
+
+Creating a feature vector
+-------------------------
+
+A feature vector of a graph can be created by choosing a collection of orbits or events and
+evaluating their probabilities with respect to GBS sampling with the embedded graph. These
+probabilities are then selected to be elements of the feature vector. Evaluating the
+probabilities of orbits or events can be achieved through two approaches:
+
+- **Direct sampling:** infer the probability of orbits or events from a set of sample data.

```

Comment:
Do we want to explain mathematically precisely how this is done? Otherwise I feel it contrasts too much with the explanation for the Monte Carlo approach below

Reply:
Nice! This is what I added:

```rst
In the direct sampling approach, :math:`N` samples are taken from GBS with the embedded graph and
the number that fall within a given orbit or event :math:`E` are counted, resulting in the count
:math:`c_{E}`. The probability :math:`p(E)` of an orbit or event is then approximated as:

.. math::
    p(E) \approx \frac{c_{E}}{N}
```

Reply:
Looks good! Maybe replace "the embedded graph and the number that..." with "the embedded graph. The number that..." ?

Reply:
Thanks! Done